### DISCUSSION TITLE
About airflow.api.auth.backend.default

### DISCUSSION THREAD

**Discussion by yad1d1** [apache/airflow#22729] [Q&A] [answered]
I set `auth_backend` to `airflow.api.auth.backend.default` in /opt/airflow/airflow.cfg but it doesn't work, my airflow WEB UI still shows that login is required, what should I do to cancel the login verify? My airflow version is 2.2.1

**Comment by potiuk** [comment, marked as answer]
airflow.api.auth.backend.default is only for API. Airlfow 2 uses "RBAC" UI (Role Based Access Control) and you need to set Public user. It is not recommended, but you can configure it following links described in FAB documentation here: https://airflow.apache.org/airflow.api.auth.backend.default/apache-airflow/stable/security/access-control.html

**Reply by yad1d1** [reply]
Thank you!
